### PR TITLE
bump release versions to build 1.12 release artifacts

### PR DESCRIPTION
**Pull request by krzyzacy**
/hold
hold for there are a few other changes need to be made, also working on a doc :-)

/area jobs
/milestone 1.12
cc @cjwagner @amwat @tpepper

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,10 +108,10 @@ We test different master/node image versions against multiple k8s branches on di
 
 Those jobs are using channel based versions, current supported testing map is:
 - k8s-dev : master
-- k8s-beta : release-1.11
-- k8s-stable1 : release-1.10
-- k8s-stable2 : release-1.9
-- k8s-stable3 : release-1.8
+- k8s-beta : release-1.12
```
Comment (BenTheElder) [inline README.md]: does the branch even exist yet? it doesn't look like it https://github.com/kubernetes/kubernetes/branches/all

Comment (krzyzacy) [reply README.md]: it will be cut this week

File: prow/config.yaml [hunk 2]
```diff
@@ -3695,14 +3697,15 @@ periodics:
     containers:
     - image: gcr.io/k8s-testimages/bootstrap:v20180730-c3257f6c0
       args:
-      - "--repo=k8s.io/kubernetes=release-1.10"
+      - "--repo=k8s.io/kubernetes=release-1.11"
```
Comment (cjwagner) [inline prow/config.yaml]: Do we need a `"--timeout=75"` here like there is elsewhere?

File: README.md (hunk 1 above)
Comment (krzyzacy) [reply README.md]: it exists now

File: prow/config.yaml (hunk 2 above)
Comment (krzyzacy) [reply prow/config.yaml]: done

**Comment by krzyzacy** [comment]
PTAL? we probably want to get the build jobs running first :-)

then https://github.com/kubernetes/test-infra/pull/9034, and then https://github.com/kubernetes/test-infra/pull/9067

**Comment by krzyzacy** [comment]
/hold cancel
:shipit: :crossed_fingers:

**Pull request merged**